tests/licensedcode/data/additional_licenses/additional_license_combined_test.expected.json

```diff
@@ -28,6 +26,7 @@
           "matcher": "2-aho",
           "license_expression": "example-installed-2",
           "rule_identifier": "example-installed-2.LICENSE",
+          "rule_relevance": 100,
```

Comment:
IMHO we should also inline the rule length here, as the combo of 
"score", "matched_length", "match_coverage", "matcher", "rule_relevance" and "rule_length" are what matters to diagnostic detection issues (at least that's what I use)

Reply:
The initial idea was, everything that is detection related stays, everything that is a rule attribute is moved to the license references. Now there are two options:

1.  add a CLI option `inline` which would have all these details, which would facilitate detection diagnostics. 

2. Not have this at all, and we add all the details in the `--review` options, and we have the details available with `--license-reference` anyway.

I think we want to move towards workbench/scancode.io license text to diagnose licence detection issues, and this makes the case where we want to have these details inlined by default a very tiny minority, not sure we want to move away from our general direction to support this. 

What do yoh think @pombredanne?

Reply:
After discussion with @pombredanne and @DennisClark, it is concluded that we need to have:

1. lean top level license detections (no match details)
2. resource/package level details have the matches, and also the detection_identifier pointing to the top level unique detection, and not in a separate parallel list.

Reply:
This is added, done!